CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## XX.XX.XX
```

Comment:
lets add to this version instead, it seems complete, will merge

Reply:
sure